README.md

```diff
@@ -1,9 +1,8 @@
-# Subvisual Blog
-[![Build Status](https://semaphoreci.com/api/v1/projects/d4cb3462-8468-4dca-888f-9e311039ce8d/430739/shields_badge.svg)](https://semaphoreci.com/gb/blog-subvisual-co)
-[![Code Climate](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/gpa.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co)
-[![Test Coverage](https://codeclimate.com/github/subvisual/blog.subvisual.co/badges/coverage.svg)](https://codeclimate.com/github/subvisual/blog.subvisual.co/coverage)
+# Subvisual Blog (Deprecated)
 
-Our blog. Live @ [https://subvisual.co/blog](https://subvisual.co/blog)
+**Notice: This blog has since been reimplemented as a static page. Visit the new
+repo [here](https://github.com/subvisual/blog)**. Bye bye Rails, its's been
```

Comment:
Contraction, not contradiction. `It's` is the contraction of `it is` or `it has`.